packages/rancher-cis-benchmark/charts/crds/clusterscan.yaml

```diff
@@ -19,6 +19,9 @@ spec:
   - JSONPath: .status.summary.skip
     name: Skip
     type: string
+  - JSONPath: .status.summary.warn
```

Comment:
clusterScan CRD is also changed in the other PR to include recurring scans. So I'm not sure how to merge the PRs in which order, or will there be one PR to include all these changes?
@prachidamle @cbron